static/app/views/aiAnalytics/aiAnalyticsCharts.tsx

```diff
@@ -55,8 +57,15 @@ export function TotalTokensUsedChart() {
   );
 }
 
-export function NumberOfPipelinesChart() {
+interface NumberOfPipelinesChartProps {
+  groupId?: string;
+}
+export function NumberOfPipelinesChart({groupId}: NumberOfPipelinesChartProps) {
   const {selection, isReady: isGlobalSelectionReady} = usePageFilters();
+  let query = 'span.category:"ai.pipeline"';
```

Comment:
Same note about `category` here

Reply:
You can construct these a little easier, if you want:

```
const query = MutableSearch.fromQueryObject({
  'span.category': 'ai',
  'span.group': groupId
}).formatString();
```

It'll take care of handling `undefined` and escape double quotes, and so on